site/content/docs/5.3/examples/download-app/index.html

```diff
@@ -0,0 +1,373 @@
+        <div class="carousel-indicators">
+          <button type="button" data-bs-target="#phonesCarousel" data-bs-slide-to="0" class="active" aria-current="true" aria-label="Slide 1"></button>
+          <button type="button" data-bs-target="#phonesCarousel" data-bs-slide-to="1" aria-label="Slide 2"></button>
+          <button type="button" data-bs-target="#phonesCarousel" data-bs-slide-to="2" aria-label="Slide 3"></button>
+          <button type="button" data-bs-target="#phonesCarousel" data-bs-slide-to="3" aria-label="Slide 4"></button>
+          <button type="button" data-bs-target="#phonesCarousel" data-bs-slide-to="4" aria-label="Slide 5"></button>
+          <button type="button" data-bs-target="#phonesCarousel" data-bs-slide-to="5" aria-label="Slide 6"></button>
+        </div>
+        <div class="carousel-inner d-flex align-items-center pt-4 pb-5 images" style="min-height: calc(216vw - 208px);">
+          <div class="carousel-item px-5 active">
+            <img loading="lazy" src="img/Components_dark.png" alt="" class="w-100 dark-show">
+            <img loading="lazy" src="img/Components.png" alt="" class="w-100">
+          </div>
+          <div class="carousel-item px-5">
+            <img loading="lazy" src="img/Buttons_dark.png" alt="" class="w-100 dark-show">
+            <img loading="lazy" src="img/Buttons.png" alt="" class="w-100">
+          </div>
+          <div class="carousel-item px-5">
+            <img loading="lazy" src="img/Emphasis_dark.png" alt="" class="w-100 dark-show">
+            <img loading="lazy" src="img/Emphasis.png" alt="" class="w-100">
+          </div>
+          <div class="carousel-item px-5">
+            <img loading="lazy" src="img/Card_dark.png" alt="" class="w-100 dark-show">
+            <img loading="lazy" src="img/Card.png" alt="" class="w-100">
+          </div>
+          <div class="carousel-item px-5">
+            <img loading="lazy" src="img/List_dark.png" alt="" class="w-100 dark-show">
+            <img loading="lazy" src="img/List.png" alt="" class="w-100">
+          </div>
+          <div class="carousel-item px-5">
+            <img loading="lazy" src="img/Input_dark.png" alt="" class="w-100 dark-show">
+            <img loading="lazy" src="img/Input.png" alt="" class="w-100">
+          </div>
+        </div>
+        <button class="carousel-control-prev" type="button" data-bs-target="#phonesCarousel" data-bs-slide="prev">
+          <span class="carousel-control-prev-icon" aria-hidden="true"></span>
+          <span class="visually-hidden">Previous</span>
+        </button>
+        <button class="carousel-control-next" type="button" data-bs-target="#phonesCarousel" data-bs-slide="next">
+          <span class="carousel-control-next-icon" aria-hidden="true"></span>
+          <span class="visually-hidden">Next</span>
+        </button>
+      </div>
+      <div class="row row-cols-3 row-cols-md-6 gy-5 mb-4 d-none d-sm-flex images">
```

Comment:
Those images are zoomed in only with the mouse, shouldn't they be focusable to enable this zoom as well?

Reply:
Linked to the previous comment

Reply:
Not really, `aria-hidden` won't block the keyboard navigation. Talking here about someone just using the keyboard instead of the mouse.

Reply:
Debated together, we probably won't do it.